src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -6533,7 +7113,48 @@ class Program
     }
 }";
             var comp = CreateCompilationWithSpanAndMemoryExtensions(source, parseOptions: TestOptions.Regular.WithLanguageVersion(languageVersion));
-            comp.VerifyDiagnostics(
+            comp.VerifyEmitDiagnostics(
+                // (7,20): error CS8352: Cannot use variable 's1' in this context because it may expose referenced variables outside of their declaration scope
+                //         return ref s1[1]; // 1
+                Diagnostic(ErrorCode.ERR_EscapeVariable, "s1").WithArguments("s1").WithLocation(7, 20));
+        }
+
+        // A version of above with an explicit definition of Span<T>.
```

Comment:
What does the explicit definition change though? I'm not certain why this test is needed.

Reply:
The explicit definition doesn't change the behavior. I just found it useful to have a simpler test though.